index.js

```diff
@@ -1 +1,99 @@
-// implement your API here
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+
+//Post
+server.post('/api/users', (req, res) => {
+    const { name, bio, created_at, updated_at } = req.body;
+    if (!name || !bio) {
+        return res.status(400).json({ errorMessage: 'Please provide name and bio for the user.' })
+    }
+    db
+        .insert({
+            name,
+            bio,
+            created_at,
+            updated_at
+        })
+        .then(res => {
+            res.status(201).json(res);   
+        })
+        .catch(() => {
+            res.status(500).json({ error: 'There was an error while saving the user to the database.' })
+        })
+})
+
+//Get
+server.get('/api/users', (req, res) => {
+    db
+        .find()
+        .then(users => {
+            res.status(200).json(users);
+        })
+        .catch(error => {
+            res.status(500).json({ error: 'The users information could not be retrieved.' });
+        });
+});
+
+//Get (user)
+server.get('/api/users/:id', (req, res) => {
+    const id = req.params.id;
+        db
+            .findById(id)
+            .then(users => {
+                if(!user.id) {
```

Comment:
```suggestion
                if(!users) {
```
You just need to check if the user is returns truthy or falsy